packages/compass-crud/src/components/table-view/cell-editor.jsx

```diff
@@ -54,7 +58,7 @@ class CellEditor extends React.Component {
    * in this cell, get the type of the column from this.props.column and add a
    * field to the HadronDocument that is empty.
    */
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
```

Comment:
I think fixing this is probably not worth doing. Might be worth investigating using a table library before investing much more time into the table view. Open to other ideas, could lump it into COMPASS-5847 but it would blow up the scope and might introduce some unintentional bugs.